components/cosmetic_filters/resources/data/cosmetic_filters.js

```diff
@@ -0,0 +1,543 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
```

Comment:
Looks like this is a replacement for `components/brave_extension/extension/brave_extension/content_cosmetic.ts` - would be good to do a `git mv` of that file in the same commit so that we can retain history and more easily see exactly what we're changing here - this content script evolved a fair amount of nuance as we solved issues.
But more importantly also IMO we'd be a lot safer keeping on using typescript rather than deleting it as is the case here. The build target is already setup for that so you just need to use a .ts file ending and the output will still be the same bundle. Do you need help with using typescript here?